scrapy/utils/display.py

```diff
@@ -3,20 +3,56 @@
 """
 
 import sys
+from packaging.version import parse
```

Comment:
packaging is not an stdlib module, is it?

Reply:
no, it is a third party utility used by setuptools.

Reply:
I think we can use `distutils.version.LooseVersion()` or `distutils.version.StrictVersion()` for it but most users must have installed setuptools.

Reply:
@Gallaecio Does scrapy install ``setuptools`` package with other reqirements during installation?

Reply:
I don’t think so, you can check in `setup.py`. However, can’t we just [use a built-in alternative](https://stackoverflow.com/a/11887885/939364)?

Reply:
@Gallaecio thanks for the suggestion. I have changed version comparison with inbuilt function `distutils.version.LooseVersion()`.